modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -26,75 +26,24 @@ const LETTER_DIGIT_REGEX = /[A-Z0-9]/gi;
 const NOT_LETTER_REGEX = /[^a-z]/gi;
 const YEARS_INDEX = 6;
 
-const getDateMask = (dateDelimiter, dateFormat) => {
-	const lastSymbol = dateFormat.slice(-1).match(NOT_LETTER_REGEX);
-
-	dateFormat = lastSymbol ? dateFormat.slice(0, -1) : dateFormat;
-
-	return dateFormat
-		.split(dateDelimiter)
-		.map((item) => {
-			let currentFormat;
-
-			if (item === 'YYYY') {
-				currentFormat = 'yyyy';
-			}
-			else if (item === 'DD') {
-				currentFormat = 'dd';
-			}
-			else {
-				currentFormat = 'MM';
-			}
-
-			return currentFormat;
-		})
-		.join(dateDelimiter);
-};
-
-const getDelimiter = (dateFormat) => {
-	let dateDelimiter = '/';
-
-	if (dateFormat.indexOf('.') !== -1) {
-		dateDelimiter = '.';
-	}
-
-	if (dateFormat.indexOf('-') !== -1) {
-		dateDelimiter = '-';
-	}
-
-	return dateDelimiter;
-};
-
-const getLocaleDateFormat = (locale, format = 'L') => {
+const getLocaleDateFormat = (locale) => {
```

Comment:
Hi @miltonmc! Awesome changes you've made, but I have a question about this part: why did you remove the "format" parameter here? Because for exemple, on the task of the story I'm working on, I do something like this:

```
const date = momentLocale.localeData().longDateFormat('L');
const time = momentLocale.localeData().longDateFormat('LT');
```
So if the function had the "format" parameter I could pass the 'L' or 'LT' in this case, doing something like this:

```
const date = getLocaleDateFormat('L');
const time = getLocaleDateFormat('LT');
```

Wouldn't this be better or is not really necessary since it's a simple case?

Reply:
I've removed the parameter, because it was not been called anywhere.

You will probably need to add a new parameter to specify if the format will contain the time or not, but not to specify the format (as the one I've removed). And you got it right, you will need to do something like you've shown here, to get the time format.